dbchangelog.xml

```diff
@@ -27,11 +31,99 @@
         <comment>As per https://github.com/bb-mvp/kafka-pipeline/issues/32</comment>
         <createTable tableName="debezium_heartbeat">
             <column name="last_heartbeat_ts" type="datetime" defaultValue="${now}">
-                <constraints nullable="false" primaryKey="true" primaryKeyName="debezium_heartbeat_pkey" />
+                <constraints nullable="false" primaryKey="true"
+                    primaryKeyName="debezium_heartbeat_pkey" />
             </column>
         </createTable>
         <insert tableName="debezium_heartbeat">
             <column name="last_heartbeat_ts" value="${now}" />
         </insert>
     </changeSet>
+    <changeSet author="liquibase-docs" id="dropTable-transaction">
```

Comment:
Why is the author `liquibase-docs`?  I think it should be changed to `cfagaras`?

Reply:
another commit to the current PR is made.